html/elements/input/email.json

```diff
@@ -0,0 +1,58 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "input-email": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/html/Element/input/email",
+            "support": {
+              "chrome": {
+                "version_added": "5"
+              },
+              "chrome_android": {
+                "version_added": null
+              },
+              "edge": {
+                "version_added": null
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": null,
+                "notes": "This feature is not implemented yet. See <a href='https://bugzil.la/888320'>bug 888320</a> and <a href='https://wiki.mozilla.org/TPE_DOM/Date_time_input_types'>TPE DOM/Date time input types</a>."
+              },
+              "firefox_android": {
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": "10"
+              },
+              "opera": {
+                "version_added": "11"
+              },
+              "opera_android": {
+                "version_added": true
+              },
+              "safari": {
+                "version_added": null
+              },
+              "safari_ios": {
+                "version_added": "3.1",
+                "notes": "Safari doesn't do validation, but instead offers a custom 'email' keyboard, which is designed to make entering email addresses easier. Also, note that Safari mobile on iOS devices automatically applies a default style of <code>opacity</code>: 0.4 to disable textual <code><input></code> elements, including those of type 'email'. Other major browsers don't currently share this particular default style."
```

Comment:
This should be 2 notes, one about not doing validation and the other about opacity.

Also, does this note only apply to Safari iOS? If so it should say that, and not just "Safari". I just tested on Safari desktop and it did validate the address.

I didn't really understand how setting `opacity` to .4 would disable anything.

Reply:
This will also need to encode `<input>` in `<code><input></code>`, or you get this in the rendered output:

<img width="351" alt="screen shot 2018-06-21 at 2 43 59 pm" src="https://user-images.githubusercontent.com/432915/41747081-9c273ca8-7561-11e8-93b3-6b1e43fe54a6.png">